apps/web/src/pages/api/config/[projectId]/index.ts

```diff
@@ -0,0 +1,216 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from "server/db/client";
+import NextCors from "nextjs-cors";
+import { z } from "zod";
+import { FlagService } from "server/services/FlagService";
+import { TestService } from "server/services/TestService";
+import { hashApiKey } from "utils/apiKey";
+import {
+  abbyConfigSchema,
+  getDefaultFlagValue,
+  stringifyFlagValue,
+} from "@tryabby/core";
+import {
+  transformClientFlagToDBType,
+  transformDBFlagTypeToclient,
+} from "lib/flags";
+import { AbbyConfigFile } from "@tryabby/core";
+
+const incomingQuerySchema = z.object({
+  projectId: z.string(),
+});
+
+export default async function handler(
```

Comment:
this whole thingy is looking terrible. nested into the 1000th layer over hundreds of lines.

maybe we can use the beauty of modern programming languages: functions.

at least the individual "methods" GET/PUT..? and maybe even further..

Reply:
I agree, makes sense to break this up. Newer next versions actually allowing exporting individual functions such a `GET` or `PUT` which makes it much cleaner